docs/Plugins.md

```diff
@@ -18,6 +18,8 @@ The optional `options` parameter for `fastify.register` supports a predefined se
 + [`logLevel`](Routes.md#custom-log-level)
 + [`logSerializers`](Routes.md#custom-log-serializer)
 + [`prefix`](Plugins.md#route-prefixing-options)
++ [`middie`]()
++ [`fastify-express`]()
```

Comment:
Just like this right and then define here.

Reply:
> 2. middie and fastify-express as listed inside the core plugins.

What about this?

Reply:
This two line is not needed.

Reply:
Done!